api/v1alpha1/etcd_types.go

```diff
@@ -69,6 +69,19 @@ type CompressionPolicy string
 // +kubebuilder:validation:Enum=periodic;revision
 type CompactionMode string
 
+type CopySpec struct {
+	// SourceStore defines the specification of the source object store provider for copying backups.
+	// +optional
+	SourceStore *StoreSpec `json:"sourceStore,omitempty"`
+	// MaxAge defines the maximum age in days that one snapshot must be in order to be copied.
+	// By default all snapshots will be copied.
+	// +optional
+	MaxAge *uint32 `json:"maxAge,omitempty"`
```

Comment:
Let's make this consistent with the naming in `etcd-backup-restore`
```suggestion
	MaxSnapshotAge *uint32 `json:"maxSnapshotAge,omitempty"`
```